quartz-core/src/main/java/org/quartz/impl/jdbcjobstore/JobStoreSupport.java

```diff
@@ -3575,6 +3587,17 @@ protected void clusterRecover(Connection conn, List<SchedulerStateRecord> failed
                                     .updateTriggerStatesForJobFromOtherState(
                                             conn, jKey,
                                             STATE_PAUSED, STATE_PAUSED_BLOCKED);
+
+                            if (pauseTriggeredJobOnNodeFail) {
+                                if (ftRec.getJobKey() != null) {
+                                    log.warn(String.format("job %s paused, because it was running when node %s failed. Make sure that job is NOT running, then unpause it", ftRec.getJobKey(), ftRec.getFireInstanceId()));
+
+                                    List<OperableTrigger> triggers = getTriggersForJob(conn, ftRec.getJobKey());
+                                    for (OperableTrigger trigger : triggers) {
+                                        pauseTrigger(conn, trigger.getKey());
```

Comment:
then it will be inconsistent state: some triggers paused, some not. Also then we will have same job running at same time on diff nodes. So better to rollback full transaction, as it now?